codes.json

```diff
@@ -476,22 +476,41 @@
   }
   ,
   {
-    "title" : "ALaDyn/PICCANTE",
-    "subtitle" : "TBD/",
-    "description" : "TBD/",
+    "title" : "ALaDyn",
+    "subtitle" : "A High-Accuracy PIC Code for the Maxwell-Vlasov Equations",
+    "description" : "ALaDyn is a fully 3D electromagnetic Particle-in-cell Fortran code. It implements the selfconsistent dynamics of lasers and beams propagation in an underdense plasma and the interaction with solid targets. It is equipped with an explicit envelope solver for the laser dynamics and with a Eulerian fluid equation scheme for the background cold plasma.",
+    "application": [],
+    "compute": [],
```

Comment:
While I have you here, does ALaDyn have:
- MPI
- OpenMP
- GPU
capabilities? I think yes, yes, no, right?

Reply:
No, MPI only. We never introduced OpenMP.